### PR TITLE
Init Arabic Localization

### PR DESCRIPTION
**Pull request by mhamdisemah**
Resolve K8S Arabic Localization on this #20343, because it has lot of problems .
@sftim @mohamedsgap

### CODE REVIEW

File: content/ar/docs/setup/_index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+reviewers:
+- mohamedsgap
+- mohamedkhaledhafez
```
Comment (mohamedsgap) [inline content/ar/docs/setup/_index.md:4]:
```suggestion
```

File: content/ar/docs/setup/_index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- Benwa-98
```
Comment (mohamedsgap) [inline content/ar/docs/setup/_index.md:5]:
```suggestion
```

**Comment by mohamedsgap** [comment]
#19758 & #20343 will help you a lot to finish this PR, because they contains the rest of required contents that are needed to be able to merge this one, so we can move fast, @mhamdisemah

**Comment by sftim** [comment]
Next step: @mohamedsgap / @mustafaAnwar2020 does this look /lgtm to you?

File: content/ar/docs/tutorials/kubernetes-basics/_index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+---
```
Comment (sftim) [inline content/ar/docs/tutorials/kubernetes-basics/_index.md:1]: This file should have a `.html` file extension I think, not `.md`.

File: OWNERS_ALIASES [hunk 4]
```diff
@@ -234,3 +234,10 @@ aliases:
     - idvoretskyi
     - MaxymVlasov
     - Potapy4
+  sig-docs-ar-owners: # Admins for Arabic content
+    - mohamedsgap
+    - mhamdisemah
+  sig-docs-ar-reviews: # PR reviews for Arabic content
+    - mohamedsgap
+    - mustafaAnwar2020
```
Comment (mohamedsgap) [inline OWNERS_ALIASES:242]:
```suggestion
```

Comment (mohamedsgap) [inline OWNERS_ALIASES:242]:
```suggestion
```

File: content/ar/docs/home/_index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
+approvers:
+- chenopis
```
Comment (mohamedsgap) [inline content/ar/docs/home/_index.md:3]:
```suggestion
- mohamedsgap
```

File: i18n/ar.toml [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+other = "تحرير هذه الصفحة"
+
+[main_page_history]
+other ="محفوظات الصفحة"
+
+[main_page_last_modified_on]
+other = "آخر تعديل للصفحة في"
+
+[main_by]
+other = "by"
```
Comment (mohamedsgap) [inline i18n/ar.toml:100]:
```suggestion
other = "عن طريق"
```

File: content/ar/community/static/cncf-code-of-conduct.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+Instances of abusive, harassing, or otherwise unacceptable behavior in Kubernetes may be reported by contacting the [Kubernetes Code of Conduct Committee](https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. For other projects, please contact a CNCF project maintainer or our mediator, Mishi Choudhary <mishi@linux.com>.
+
+This Code of Conduct is adapted from the Contributor Covenant
+(http://contributor-covenant.org), version 1.2.0, available at
+http://contributor-covenant.org/version/1/2/0/
+
+### CNCF Events Code of Conduct
+
+CNCF events are governed by the Linux Foundation [Code of Conduct](https://events.linuxfoundation.org/code-of-conduct/) available on the event page. This is designed to be compatible with the above policy and also includes more details on responding to incidents.
```
[the remark this answers is not in the document]
Comment (mhamdisemah) [reply content/ar/community/static/cncf-code-of-conduct.md]: @sftim yes i think it is better to use right-to-left rendering with arabic  , i will have a look how to establish this , if you have any idea tell us .

**Comment by mohamedsgap** [comment]
@sftim @mhamdisemah about rendering the content from left to right or right to left, in Arabic it must render from right to left, but markdown still not support the arabic language, for best practices in the arabic translation, we gonna translate all the content of the pages to Arabic, but in the page that has code snippets and some explanation it gonna be hard to translate and also be confusing for the Arabic user, so we should leave it as it is! so this is what I think and I need to hear your opinion too

**Comment by mohamedsgap** [comment]
Hey fellow, @mhamdisemah Ramadan Kareem :smile: 
any update ?

**Comment by kbhawkey** [comment]
Hello @mhamdisemah .
I am checking whether you are actively working on this pull request?

**Comment by sftim** [comment]
@mhamdisemah - how's this work going?

Did you see the new Slack channel, https://kubernetes.slack.com/messages/sig-docs-localizations ?

**Comment by kbhawkey** [comment]
Hello @mhamdisemah .
I am checking whether you are actively working on this pull request?
Would you have time to rebase your changes?

**Comment by kbhawkey** [comment]
Hello @mhamdisemah .
I am closing this pull request.
When you are ready to work on these changes, feel free to reopen this PR.
/close

**Comment by mboukhalfa** [comment]
@mohamedsgap How should we proceed with this PR?

**Pull request closed without merging**